molgenis-data/src/main/java/org/molgenis/data/RepositoryCollection.java

```diff
@@ -75,6 +75,8 @@
 	 */
 	void deleteRepository(EntityType entityType);
 
+	void updateRepository(EntityType entityType, EntityType updatedEntityType);
```

Comment:
Perhaps make a bit clearer that this is only called for updates to EntityType's own properties, and not changes to one or more of its Attributes